js/TextPane.js

```diff
@@ -20,7 +20,7 @@ class TextPane extends React.Component {
     switch (e.type) {
       case 'keydown':
       case 'keypress':
-        e.preventDefault();
+//        e.preventDefault();  /// this would prevent entering text into input box
```

Comment:
IIRC this was trying to prevent text panes from submitting on enter. Plus, the goal is to send keypress events to the backend, not necessarily to have them render up-front here.

Reply:
Ah I see. I was putting forms into visdom window, and .preventDefault would make the form inputting fairly laggy, hence commented out.